purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -228,71 +230,31 @@ class PostingTransactionsTests {
     }
 
     @Test
-    fun `store user id is sent when posting to backend`() {
```

Comment:
I see you removed this to move it to the Amazon tests, can you move it there? Otherwise I think we'll forget and lose this test

Reply:
there actually already was one